README.md

```diff
@@ -1 +1,61 @@
-# ironfoudation-team-project
+# WOOF application 
+
+![macOS](https://img.shields.io/badge/macOS-13.3+-blue) ![Swift](https://img.shields.io/badge/Swift-5.8-orange)  ![Xcode](https://img.shields.io/badge/Xcode-14.3-red) ![SwiftFormat](https://img.shields.io/badge/SwiftFormat-0.51.7-white) ![SwiftLint](https://img.shields.io/badge/SwiftLint-0.51.0-green)
+
+## Table of contents
+* [General info](#general-info)
+* [Technologies](#technologies)
+* [Installation](#installation)
+* [Authors](#authors)
+* [Notes](#notes)
+
+## General info
+
+This is the application for iOS.
+Woof application connects dog owners with pet sitters which contributes to build trusted relationships between them. The app makes it easy for nearby pet owners to find pet sitting and makes convienience of responding to jobs for pet sitters from phone.
+App also makes managing sitting bussines and pet owner's schedule. They can choose their own schedule, prices, and requirements for servise. 
+
+**It also important to know:**
+To process the application some personal information about customers is required, for example, home address, mobile phone.
+
+## Technologies
+
+To contribute code, please, use:
+* macOS minimal version - Ventura 13.3
+* Xcode version: 14.3
+* Swift: 5.8
+* SDKs for iOS: 15.0+
+* SwiftLint: 0.51.0: [SwiftLint](https://github.com/realm/SwiftLint)
+* SwiftFormat: 0.51.7: [SwiftFormat](https://github.com/nicklockwood/SwiftFormat)
+
+## Installation
+
+ Supported devices - iPhone only.
+ 
+**Pre-requisites**: 
+To run this project: install Xcode (SDKs for iOS 15.0+ as a deployment target).
```

Comment:
I think, when we will clone our project, the deployment target will be set, and we don't need to specify it here.
```suggestion
- installed Xcode with the version specified under the Development Environments section.
```

Reply:
Agree, thanks. Fixed it.